src/components/atoms/OTPInput.tsx

```diff
@@ -0,0 +1,62 @@
+import { HStack, Input } from 'native-base';
+import React, { useRef, useState } from 'react';
+import { View, TouchableWithoutFeedback } from 'react-native';
+
+interface OTPProps {
+  count?: number;
+  required?: boolean;
+}
+
+interface NativeBaseInputProps {
+  focus: () => void;
+  isFocused: () => boolean;
+}
+
+const OTPInput: React.FC<OTPProps> = ({ count = 6 }) => {
+  const inputRef = useRef<typeof Input & NativeBaseInputProps>();
+  const [otpCode, setOTP] = useState<string>('');
+
+  const handleOTPInput = (text: string) => {
+    if (text.match(/^[0-9]{0,6}$/)) setOTP(text);
+  };
+
+  const focusOTP = () => {
+    inputRef?.current?.focus();
+    console.log('focus otp');
```

Comment:
consider remove console.log

Reply:
oh I forgot. Thanks for pointing that out